config.json

```diff
@@ -464,6 +464,18 @@
         "str_vs_string"
       ]
     },
+    {
+      "slug": "rail-fence-cipher",
+      "uuid": "9a8bae4f-2c0b-4e9e-aab2-b92f82dd3b87",
+      "core": false,
+      "unlocked_by": "atbash-cipher",
+      "difficulty": 4,
+      "topics": [
+        "chars",
+        "cipher",
```

Comment:
perhaps it would be useful to have a separate PR add that topic to the other cipher exercises.

That is only important if there's a filter-by-topic functionality on the website, and I'm not yet sure there is such a one.